Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -580,6 +583,13 @@ open class ComposerVC: _ViewController,
                     guard let `self` = self else { return }
                     self.sendRedPacketAction()
                 }
+            case .gift:
+                self.animateToolkitView(isHide: true)
+                self.composerView.inputMessageView.textView.resignFirstResponder()
+                DispatchQueue.main.asyncAfter(deadline: .now() + 0.3) { [weak self] in
```

Comment:
@tunguyen-zien  can we remove weak self from here, we already have weak self in top level of closures.

Reply:
i have updated it. Thanks @parth-simformsolutions